source/versions/history.rst

```diff
@@ -1,6 +1,47 @@
 Package History
 ===============
 
+4.3.2 (2020-10-15)
+------------------
+
+Major Features and Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Add auto entity annotating tool which leverages language models like BERT to add entity annotations to training queries
```

Comment:
Does spacy use BERT?

Reply:
I don't think so. Even if it does, it'd be better to keep this more generic and just say "Add auto entity annotating tool which leverages pre-trained NLP models to add entity annotations to training queries"

Reply:
Done